### PR TITLE
Ensure dates are stored and received in UTC

### PR DESCRIPTION
**Pull request by jaceko**

### CODE REVIEW

File: example-context/example-service/example-liquibase/src/main/resources/liquibase/view-store-db-changesets/003-order-table.changelog.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+               logicalFilePath="003-order-table.changelog.xml">
+
+        <createTable tableName="cake_order">
+            <column name="order_id" type="uuid">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="recipe_id" type="uuid">
+                <constraints nullable="false"/>
+            </column>
+            <column name="delivery_date" type="datetime">
```
Comment (purple52) [inline example-context/example-service/example-liquibase/src/main/resources/liquibase/view-store-db-changesets/003-order-table.changelog.xml]: I think this should be "datetime tz"

Comment (jaceko) [inline example-context/example-service/example-liquibase/src/main/resources/liquibase/view-store-db-changesets/003-order-table.changelog.xml]: fixed

File: common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java [hunk 2]
```diff
@@ -98,7 +117,7 @@ private JsonObject jsonObject() {
         private final InternalPojo internalPojo;
 
 
-        public Pojo(UUID id, String name, List<String> attributes, InternalPojo internalPojo) {
+        public Pojo(@JsonProperty("id") UUID id, @JsonProperty("name") String name, @JsonProperty("attributes") List<String> attributes, @JsonProperty("internalPojo") InternalPojo internalPojo) {
```
Comment (purple52) [inline common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java]: Do we need these? The objectMapper should be able to get the names from the JDK8 parameter names.

File: common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java [hunk 3]
```diff
@@ -127,7 +146,7 @@ public InternalPojo getInternalPojo() {
         private UUID internalId;
         private String internalName;
 
-        public InternalPojo(final UUID internalId, final String internalName) {
+        public InternalPojo(final @JsonProperty("internalId") UUID  internalId, final @JsonProperty("internalName") String internalName) {
```
Comment (purple52) [inline common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java]: Do we need these? The objectMapper should be able to get the names from the JDK8 parameter names.

File: common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java (hunk 2 above)
Comment (jaceko) [inline common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java]: fixed

**Pull request merged**